packages/discovery/src/index.ts

```diff
@@ -0,0 +1,56 @@
+export type ProviderDetail = {
+  _tag: string
```

Comment:
Is there a good reason for `_tag` (ie the underscore prefix) and not `tag` or `kind`? (Personally I prefer kind because this thing literally designates what _kind_ of interface it is)

Reply:
Sure lets go with `kind`